### PR TITLE
update bug report template

### PR DESCRIPTION
**Pull request by underhood**
##### Summary
Adds info about `-W buildinfo` flag which provides much more detail about agent version and capabilities. This will let us know in the future things like ACLK impl. used etc.

##### Component Name
github template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ Place the output from the command in the code section below.
 
 ##### Netdata version
 <!--
+Provide output of `netdata -W buildinfo`.
+
+In case your agent is older and doesn't support buildinfo do following:
```
Comment (joelhans) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]:
```suggestion
If your Netdata Agent is older, and doesn't support buildinfo, do the following:
```

Comment (underhood) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thank you :)

Comment (ilyam8) [reply .github/ISSUE_TEMPLATE/bug_report.md]: > do the following

update Netdata and re-test 😄

**Review by ilyam8 — APPROVED** [review]

**Review by joelhans — DISMISSED** [review]

**Comment by ilyam8** [comment]
@underhood please rebase, that docker build ci check is required..

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**